onadata/apps/messaging/permissions.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Tests Messaging app implementation.
+"""
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from rest_framework import exceptions, permissions
+
+
+class TargetObjectPermissions(permissions.BasePermission):
+    """
+    Check target object permissions
+    """
+    perms_map = {
+        'GET': ['%(app_label)s.view_%(model_name)s'],
```

Comment:
This is so that we are consistent with Django Rest Framework, see: https://github.com/encode/django-rest-framework/blob/c674687782ef96a3bb8466b412e99debd3d04f00/rest_framework/permissions.py#L89